library/std/src/os/wasi/io/raw.rs

```diff
@@ -2,4 +2,20 @@
 
 #![unstable(feature = "wasi_ext", issue = "71213")]
 
+// NOTE: despite the fact that this module is unstable,
+// stable Rust had the capability to access the stable
+// re-exported items from os::fd::raw through this
+// unstable module.
+// In PR #95956 the stability checker was changed to check
+// all path segments of an item rather than just the last,
+// which caused the aforementioned stable usage to regress
+// (see issue #99502).
+// As a result, the items in os::fd::raw were given the
+// rustc_allowed_through_unstable_modules attribute.
+// If new items are added to os::fd::raw and you want
```

Comment:
I don't think we should be encouraging new allowed_through_unstable items, especially because the existence of these exceptions essentially tie our hands on stabilizing their module path as is. Is there any reason to keep this module unstable at this point other than momentum?